lib/TypeShrinking/BitLiveness.cpp

```diff
@@ -33,17 +33,17 @@ struct BitLivenessAnalysis {
   using GraphType = GenericGraph<DataFlowNode> *;
   using LatticeElement = uint32_t;
   using Label = DataFlowNode *;
-  using MFPResult = MFPResult<BitLivenessAnalysis::LatticeElement>;
+  using MFPResult = MFP::MFPResult<BitLivenessAnalysis::LatticeElement>;
 
-  static uint32_t combineValues(const uint32_t &Lh, const uint32_t &Rh) {
+  uint32_t combineValues(const uint32_t &Lh, const uint32_t &Rh) const {
     return std::max(Lh, Rh);
   }
 
-  static bool isLessOrEqual(const uint32_t &Lh, const uint32_t &Rh) {
+  bool isLessOrEqual(const uint32_t &Lh, const uint32_t &Rh) const {
     return Lh <= Rh;
```

Comment:
`sed s/Lh/LHS/`
`sed s/Rh/RHS/`

For coherence with other parts of the codebase.